app/helpers/organisation_helper.rb

```diff
@@ -24,4 +24,10 @@ def search_results_to_documents(search_results, organisation, include_metadata:
       end,
     }
   end
+
+  def array_of_links_to_organisations(organisations)
```

Comment:
I think this is probably the wrong place for this method. When collections thinks about Organisations in this context it means pages listed here https://www.gov.uk/government/organisations

I think this method is in the organisation helper in whitehall because it's actually used by the [organisation frontend here](https://github.com/alphagov/whitehall/blob/5cba22ae73efabd6e93f8da75b28a2ae88ffdec7/app/views/organisations/_organisations_name_list.html.erb#L4-L6). Because we're not using this method in collections organisation view and it would be limited to topical events this is probably better placed in the topical events controller